src/create-parts-v2.js

```diff
@@ -16,18 +16,21 @@ const seen = new Set()
 const sleep = ms => new Promise(resolve => setTimeout(resolve, ms))
 
 const lambdaName = process.env.DUMBO_CREATE_PART_LAMBDA
+// invokes lambda fucntion to create the car parts?
 const createPartsRequest = async (opts, retries = 5) => {
```

Comment:
Let’s remove this whole function. The lambda call itself does retries, this was a temporary hack for some other infra errors we were seeing.